modules/calcite/src/test/sql/function/string/test_space.test

```diff
@@ -0,0 +1,9 @@
+# name: test/sql/function/string/space.test
+# description: Test the SPACE function
+# group: [string]
+
+query II
+select sha1('hello'), sha1(NULL)
```

Comment:
this query belongs to `sha1.test` script

Reply:
fixed